terraform/environments/core-shared-services/instance-scheduler-lambda-function.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_lambda_function" "instance-scheduler-lambda-function" {
+  #checkov:skip=CKV_AWS_116
+  #checkov:skip=CKV_AWS_117
+  #checkov:skip=CKV_AWS_272 "Code signing not required"
+  function_name                  = "instance-scheduler-lambda-function"
+  handler                        = "main"
+  reserved_concurrent_executions = 0
+  runtime                        = "go1.x"
```

Comment:
can we lock the version more? what do we use in other scripts?

Reply:
I don't think we can. According to https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html#SSS-CreateFunction-request-Runtime at the moment `go1.x` seems to be the only valid value for the runtime configuration for the Go language.